index.html

```diff
@@ -432,6 +432,75 @@ <h3>Obtaining Consent</h3>
         </li>
       </ol>
     </section>
+    <section>
+      <h3 id=permissions-integration>Permissions Integration</h3>
+      <p>The Audio Output Devices API is a [=powerful feature=] that is
+      identified by the [=powerful feature/name=] "speaker-selection".</p>
+      <p>It defines the following types and algorithms:</p>
+      <dl>
+        <dt>
+          [=powerful feature/permission descriptor type=]
+        </dt>
+        <dd>
+          <p>
+            A permission covers access to the device given in the associated
+            {{DevicePermissionDescriptor}} descriptor.
+          </p>
+          <p>
+            If the descriptor does not have a {{DevicePermissionDescriptor/deviceId}}, its
+            semantic is that it queries for access to all devices of that class. Thus, if a query
+            for the "speaker-selection" [=powerful feature=] with no
+            {{DevicePermissionDescriptor/deviceId}} returns {{PermissionState/"granted"}}, the
+            client knows that there will never be a permission prompt for an audio output device,
```

Comment:
Never is a strong word, and this sentence is confusing, as *selectAudioOutput* uses a picker, not a "permission prompt". Its design requires JS to have previous knowledge of a *deviceId* to skip a prompt, which it can only obtain from the user picking its device in a prompt. There is no collateral exposure.